docs/docs-beta/docs/dagster-plus/access/rbac/users.md

```diff
@@ -1,7 +1,88 @@
 ---
-title: "User management"
+title: "Managing users in Dagster+"
 displayed_sidebar: "dagsterPlus"
-sidebar_position: 1
+sidebar_label: "User management"
+sidebar_position: 10
 ---
 
-# User management in Dagster+
\ No newline at end of file
+Dagster+ allows you to grant specific permissions to your organization's users, ensuring that Dagster users have access only to what they require.
```

Comment:
This guide focuses only on the actual managing of users. Information about user roles and permissions will be added in a follow up PR.

Reply:
Would it be all on one page? I can imagine a page called 'Managing Users, Roles, and Permissions' being pretty useful and succinct.

Reply:
I was concerned about that page getting super long, which is what we told everyone else not to worry about 😅 I'll see what I can do.